test/src/sender/main.cpp

```diff
@@ -23,119 +23,164 @@
 
 using namespace std;
 
-using hclock=chrono::high_resolution_clock;
+using hclock = chrono::high_resolution_clock;
 using TIME = NDTime;
 
 
 /***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp_controll : public cadmium::in_port<Message_t>{};
-struct inp_ack : public cadmium::in_port<Message_t>{};
+struct inp_controll : public cadmium::in_port<Message_t> {};
+struct inp_ack : public cadmium::in_port<Message_t> {};
 
 /***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp_ack : public cadmium::out_port<Message_t>{};
-struct outp_data : public cadmium::out_port<Message_t>{};
-struct outp_pack : public cadmium::out_port<Message_t>{};
+struct outp_ack : public cadmium::out_port<Message_t> {};
+struct outp_data : public cadmium::out_port<Message_t> {};
+struct outp_pack : public cadmium::out_port<Message_t> {};
 
 
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
 template<typename T>
 class ApplicationGen : public iestream_input<Message_t,T> {
-public:
-  ApplicationGen() = default;
-  ApplicationGen(const char* file_path) : iestream_input<Message_t,T>(file_path) {}
+    public:
+      ApplicationGen() = default;
+        ApplicationGen(const char* file_path) : 
+            iestream_input<Message_t,T>(file_path) {
+    }
 };
 
 
 int main(){
```

Comment:
I would suggest to give a space between ) { to have uniformity in code
i.e., int main( ) {

Reply:
solved